receiver/jmxreceiver/receiver.go

```diff
@@ -70,8 +73,11 @@ func (jmx *jmxMetricReceiver) Start(ctx context.Context, host component.Host) (e
 
 	subprocessConfig := subprocess.Config{
 		ExecutablePath: "java",
-		Args:           append(jmx.config.parseProperties(), "-jar", jmx.config.JARPath, "-config", "-"),
+		Args:           append(jmx.config.parseProperties(), jmxMainClass, "-config", "-"),
 		StdInContents:  javaConfig,
+		EnvironmentVariables: map[string]string{
+			"CLASSPATH": jmx.config.parseClasspath(),
```

Comment:
I believe this will always overwrite the existing env var given how the subprocess environment is built. Should this append to it instead?

Reply:
Yeah that's probably worth doing. I know it doesn't always exist but we wouldn't want to overwrite it entirely I'll do a lookup and append.